dasharo/project_config.yaml

```diff
@@ -0,0 +1,27 @@
+# Dasharo project configuration data
+
+client:
+        name: Immunefi
+
+framework:
+        name: dasharo
+        version: _
```

Comment:
@kl3mdeb It should be short hash of the commit we base our development on. Please check the first commit before 1st change of @miczyg1 in the git history: https://github.com/Dasharo/coreboot/commits/asus_kgpe-d16/develop